utils/Logger/Logger.test.ts

```diff
@@ -8,19 +8,22 @@ const logger = {
   warn: jest.fn(),
 } as Record<keyof Console, jest.Mock>
 
+/* eslint-disable @typescript-eslint/no-empty-function */
 jest.mock('console', () => ({
   __esModule: true,
   Console: class ConsoleMock {
     constructor() {}
 
-    warn = logger.warn;
-    info = logger.info;
-  }
+    warn = logger.warn
+    info = logger.info
+  },
 }))
+/* eslint-enable @typescript-eslint/no-empty-function */
```

Comment:
bof bof (english)

Reply:
It is a typo defined by eslint https://eslint.org/docs/2.13.1/user-guide/configuring